crates/bevy_ecs/src/observer/mod.rs

```diff
@@ -536,6 +536,8 @@ impl World {
     /// Spawns a "global" [`Observer`] which will watch for the given event.
     /// Returns its [`Entity`] as a [`EntityWorldMut`].
     ///
+    /// `system` can be any system whose first parameter is a [`Trigger`].
```

Comment:
Nit: Maybe we want to rename the parameter here to observer to match the rest of the codebase?